.github/workflows/publish_build.yml

```
@@ -0,0 +1,84 @@
+            ${{ steps.changelog.outputs.changelog }}
+          draft: false
+          prerelease: false
+          commitish: dev
+      - name: Build Distribution Packages
+        run: |
+          python setup.py bdist_wheel
+      - name: Prepare next Build version
+        run: echo "::set-output name=version::$(python setup.py --version)"
+        id: alpha
+      - name: Increment Version ${{ steps.alpha.outputs.version }}Alpha0
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2034:warning:1:1: VER appears unused. Verify use (or export if used externally)

(shellcheck)